integ-tests/src/test/java/com/google/integration/IntegrationTests.java

```diff
@@ -80,5 +86,31 @@ public void usersTest() throws SQLException, IOException {
           AvroHelper.getLongNotNull(record, "LastAccessTimeStamp");
       avroList.add(avroRow);
     }
+
+    List<UserTableRow> dbListCopy = new ArrayList<>(dbList);
+    avroList.forEach(dbList::remove);
+    dbListCopy.forEach(avroList::remove);
+
+    if (dbList.size() == 0 && avroList.size() == 0) {
+      logger.info("DB view and Avro file are equal");
+    } else if (dbList.size() > 0 && avroList.size() > 0) {
+      logger.error("DB view and Avro file have mutually exclusive row(s)");
+
+      logger.info(format("DB view '%s' has %d different row(s)", sql, dbList.size()));
+      dbList.forEach(System.out::println);
```

Comment:
Is this debugging output? If this is meant to stay, shouldn't this rather use the same logger as above?

Reply:
Yep, valid point, but there is a small inconvenience:
1. Logger standard message looks like -
Jan 10, 2022 7:22:05 PM com.google.integration.IntegrationTests usersTest
INFO: DB view 'src/main/java/com/google/sql/users.sql' has 16 different row(s)
2. Meanwhile println put everything line by line -
{UserName: {string: DBC}, CreatorName: {string: DBC}, CreateTimeStamp: {long: 1503419580000}, LastAccessTimeStamp: {long: 0}}
{UserName: {string: All}, CreatorName: {string: DBC}, CreateTimeStamp: {long: 1503419580000}, LastAccessTimeStamp: {long: 0}}

I can adjust the CLI output format for SLF4J but it will make a message really long.
If you are OK with this, then I'll do it.

Reply:
Having a long message is fine.

Reply:
Done.